nixos/modules/services/databases/redis.nix

```diff
@@ -224,6 +229,16 @@ in
 
     environment.systemPackages = [ cfg.package ];
 
+    systemd.services.disable-transparent-huge-pages = {
+      enable = config.services.redis.enable;
+      description = "Disable Transparent Huge Pages (required by Redis)";
+      after = [ "sysinit.target" "local-fs.target" ];
```

Comment:
as long as we don't set `DefaultDependencies=no`, it already depends on `sysinit.target`, and `local-fs.target` became part of `sysinit.target` in https://github.com/NixOS/nixpkgs/pull/61321, so this line can be dropped.

Reply:
fa49f7ce6bba804a65ed61d5f7b989dfc3e084e2